lib/gke-overrides.libsonnet

```diff
@@ -28,73 +42,208 @@ local ignore_dashboards = [
   'scheduler.json',
 ];
 
+// Define a mapping of alert/recordname to expression.
+// Overrides the expr field for the specifeid record name.
+local expr_overrides = {};
+
+// Define a mapping of alertname to label page:false
+// It will add a label `page:false` else it will add a `page:true`
+local page_false_critical = [
+  'AlertmanagerConfigInconsistent',
+  'ContainerCombinedIoHighOverTime',
+  'ClusterLowMemoryAvailable',
+  'ElasticsearchClusterHealthYELLOW',
+  'ElasticsearchHeapTooHigh',
+  'ElasticsearchLowDiskFree',
+  'HAProxyFrontendSessionUsage',
+  'HAProxyBackendResponseErrors',
+  'HAProxyBackendDown',
+  'HAProxyServerInBackendUpPercentageLow',
+  'KubePersistentVolumeInodeUsageCritical',
```

Comment:
Should we clarify what critical means here? If we don't page, it's can't be deemed as critical...but `Critical` is part of the alert-name (I guess its just worth describing why we wouldn't page on this)

Reply:
do you want to change the name ? 

We don't page on this because we only page on the "prediction rule" but during the day it still good got a p1 out of this , and a notification on your mobile, because is a pretty nasty situation

Reply:
Nah, if its important its important - so probably still deserve the critical name